src/components/frame/LoadingTemplate/LoadingTemplate.tsx

```diff
@@ -35,7 +35,7 @@ const MSG = defineMessages({
 });
 
 const delayedLoadingDuration = 15 * 1000; // 15 seconds
-export const failedLoadingDuration = 30 * 1000; // 30 seconds
+export const failedLoadingDuration = 3000 * 1000; // 30 seconds
```

Comment:
3000 * 1000 isn't 30 seconds.

Reply:
This was for testing, I will revert that